### DISCUSSION TITLE
Adding a dataset on hugging face?

### DISCUSSION THREAD

**Discussion by Sakshamrzt** [embeddings-benchmark/mteb#324] [Q&A] [answered]
As stated [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#a-detailed-example) we might end up changing the original dataset to be compatible with the benchmark. In such case do we add the new dataset on huggingface under [mteb](https://huggingface.co/mteb) or adding under our own user name is sufficient?

**Comment by KennethEnevoldsen** [comment]
You can supply your own user, in the future we might re-upload it under MTEB (as long as it has the right licenses) to ensure stability.

**Comment by Muennighoff** [comment, marked as answer]
Also, I'd say that everyone who contributes can also join the HF & GitHub orgs if they want to. I think on HF you can just send a request and then someone will add you as Contributor. Let me know if someone disagrees :)

**Reply by KennethEnevoldsen** [reply]
I think that is totally fine as well.

**Reply by Sakshamrzt** [reply]
Thanks for the quick answer!
However, the option to join the org on HF seems to be not activated. AFAIU the admin can enable this feature. @KennethEnevoldsen please enable if we are allowing contributors to join. Thanks!

**Reply by KennethEnevoldsen** [reply]
I have enabled the "requests to join".

**Reply by Sakshamrzt** [reply]
Thanks!

**Comment by rsgrewal-aws** [comment]
in addung your own data set -- the example provided does not work - it fails with pydantic error saying language en does not exist

**Reply by KennethEnevoldsen** [reply]
Hi @rsgrewal-aws, we recently changed to a new language standard to use the iso 639-3 language codes instead of the two letter codes (as these don't cover all languages). Additionally since languages can be in different scripts we also add a script code. Thus for english you would get "eng-Latn", English in Latin script. Hope it helps!
